packages/edit-site/src/store/actions.js

```diff
@@ -5,6 +5,11 @@ import { controls } from '@wordpress/data';
 import { apiFetch } from '@wordpress/data-controls';
 import { getPathAndQueryString } from '@wordpress/url';
 
+/**
+ * Internal dependencies
+ */
+import { store as editSiteStore } from './';
+
 /**
```

Comment:
I believe that this might cause circular reference errors - and might also be the culprit for some errors in the build - importing `store` definition inside a `store/*.js` file (this file depends on `index.js`, while `index.js` depends on this file... and so on).
In this case, you can use the same approach I used [here](https://github.com/WordPress/gutenberg/pull/28044/commits/eb9e5fd675219410a87015b9cb4387b04c18ef85#diff-d9e0955a21d8822423ca4d57842c4c7caaa54daf97bd88c950e0bb87aff80123).

Reply:
Looks like I already pushed a few commits while you were reviewing! I used your PRs for inspiration and used the approach that you mentioned! Thanks for reviewing it and great work with the rest of them! 👍

Reply:
This one is still valid. It is something that exists in several places but we should seek ways to eliminate it, ideally with an ESLint rule. Node.js / webpack deal well with this but there is this interesting cycle here:
`actions.js` imports `store` from `index.js` that imports `actions` from `actions.js`. In other places, we break the cycle by introducing another file that exports a constant with the name of the store. If you have better ideas on how to address it, we would love to apply it to other places as well :)

Reply:
Gotcha! Misunderstood it for the first time. Thanks for explaining. Pushed a commit to remove the circular dependency.

Using a separate file to store the store name sounds reasonable to me.